express/parsers/apps/espresso/parser.py

```diff
@@ -227,3 +227,12 @@ def reaction_energies(self):
         """
         neb_dat_file = os.path.join(self.work_dir, NEB_DAT_FILE)
         return self.txt_parser.reaction_energies(self._get_file_content(neb_dat_file))
+
+    def _get_esm_file(self):
+        return find_file(".esm1", self.work_dir)
```

Comment:
We might want to use prefix here too to make it safer

Reply:
We do not need as we search the whole working directory for the file. As long as it is unique it should be fine. I checked the XML file per our discussion and it seems the prefix is not stored there.